synthesizer/process/src/finalize.rs

```diff
@@ -160,9 +164,13 @@ fn finalize_fee_transition<N: Network, P: FinalizeStorage<N>>(
     fee: &Fee<N>,
 ) -> Result<Vec<FinalizeOperation<N>>> {
     // Construct the call graph.
-    let mut call_graph = HashMap::new();
-    // Insert the fee transition.
-    call_graph.insert(*fee.transition_id(), Vec::new());
+    let call_graph = if state.block_height() >= N::CONSENSUS_V3_HEIGHT {
```

Comment:
Is there any realistic possibility this can use `CONSENSUS_V2` is the chain expected to pass that block by the time this fix is applied?

Reply:
I don't think so, changes for `CONSENSUS_V2` were locked in as of last week.